src/api/java/mekanism/api/chemical/gas/attribute/GasAttributes.java

```diff
@@ -116,7 +117,7 @@ public List<Component> addTooltipText(List<Component> list) {
             ITooltipHelper tooltipHelper = ITooltipHelper.INSTANCE;
             list.add(APILang.CHEMICAL_ATTRIBUTE_COOLANT_EFFICIENCY.translateColored(EnumColor.GRAY, EnumColor.INDIGO, tooltipHelper.getPercent(conductivity)));
             list.add(APILang.CHEMICAL_ATTRIBUTE_COOLANT_ENTHALPY.translateColored(EnumColor.GRAY, EnumColor.INDIGO,
-                  tooltipHelper.getEnergyPerMBDisplayShort(FloatingLong.createConst(thermalEnthalpy))));
+                  tooltipHelper.getEnergyPerMBDisplayShort((long) thermalEnthalpy)));
```

Comment:
We either need to clamp `thermalEnthalpy` in the constructor to `Long.MAX_VALUE` or we should make this use `MathUtils#clampToLong`